cleanInstalJrcc.bat

```diff
@@ -0,0 +1,14 @@
+cd jrcc-document-access-libs\
+call mvn clean install -U
+cd ..
+cd jrcc-access-spring-boot-autoconfigure\
+call mvn clean install -U
+cd ..
+cd jrcc-access-spring-boot-starter\
+call mvn clean install -U
+cd ..
+cd jrcc-access-spring-boot-sample-app\
+call mvn clean install -U
+
+call mvn spring-boot:run
```

Comment:
Would take this out and name the file make.bat so the role of it is just to install the app

Reply:
This is handy, I already did this work sort of in the Jenkins pipeline but instead I will just reference this file so that you guys can alter its build process